hw/xbox/dsp/dsp_dma.c

```diff
@@ -137,9 +175,38 @@ static void dsp_dma_run(DSPDMAState *s)
             assert(false);
         }
 
+#ifdef DEBUG
+        char space_name = '?';
+        if (mem_space == DSP_SPACE_X) {
+            space_name = 'x';
+        } else if (mem_space == DSP_SPACE_Y) {
+            space_name = 'y';
+        } else if (mem_space == DSP_SPACE_P) {
+            space_name = 'p';
+        }
+#endif
+
+        DPRINTF("dsp dma block 0x%x (%s)\n"
+                "    next-block 0x%x%s\n"
+                "    control 0x%06x: unk0:%d, unk4:%d buf 0x%x (%s) unk9:%d format 0x%x (%s) step 0x%x\n"
+                "    count 0x%X\n"
```

Comment:
I might add the size of the resulting transfer here.
Please don't merge yet, until I've decided.

Reply:
I might do this later, for now we don't need it.